src/main/java/org/mapstruct/intellij/util/MapstructUtil.java

```diff
@@ -93,7 +93,7 @@ public final class MapstructUtil {
     public static final String INHERIT_CONFIGURATION_FQN = InheritConfiguration.class.getName();
     public static final String INHERIT_INVERSE_CONFIGURATION_FQN = InheritInverseConfiguration.class.getName();
 
-    static final String MAPPINGS_ANNOTATION_FQN = Mappings.class.getName();
+    public static final String MAPPINGS_ANNOTATION_FQN = Mappings.class.getName();
```

Comment:
nit-pick: Just for better looks, can you add a separate line between public, package-private and private constants?

Reply:
I added a new line after.